apps/mailinglists/views.py

```diff
@@ -1,6 +1,17 @@
-# -*- coding: utf-8 -*-
-
 from django.shortcuts import render
+from rest_framework import viewsets
+from rest_framework.permissions import AllowAny
+
+from .models import Mailinglist
+from .serializers import MailinglistSerializer
+
+
+class MailinglistViewSet(viewsets.ReadOnlyModelViewSet):
+    permission_classes = (AllowAny,)
+    serializer_class = MailinglistSerializer
+
+    def get_queryset(self):
+        return Mailinglist.objects.filter(public=True)
```

Comment:
Could be nice to have functionality for editing this?

Using the `rest_framework.permissions.DjangoModelPermission` class to restrict access (still allows viewing without permission).
Could also allow users with permissions to see non-public lists

Reply:
Added now!